fix date format (#11577)
demisto/content: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_1_15.md

#### Scripts
##### GetFailedTasks

> - Fix the date format in the **Incident Created Date** field.

Reviewer: ```suggestion
- Fixed the format of the **Incident Created Date** field.
```

##### IncidentsCheck-Widget-IncidentsErrorsInfo

> - Fix the date format in the **Incident Created Date** field.

Reviewer: ```suggestion
- Fixed the format of the **Incident Created Date** field.
```
